src/types/onyx/OnfidoWallet.ts

```diff
@@ -0,0 +1,21 @@
+type OnfidoWallet = {
+    /** Unique identifier returned from openOnfidoFlow then re-sent to ActivateWallet with Onfido response data */
+    applicantID?: string;
+
+    /** Token used to initialize the Onfido SDK token */
+    sdkToken?: string;
+
+    /** Loading state to provide feedback when we are waiting for a request to finish */
+    loading?: boolean;
```

Comment:
This should be isLoading

```suggestion
    /** Loading state to provide feedback when we are waiting for a request to finish */
    isLoading?: boolean;
```